parser/parser.go

```diff
@@ -160,9 +175,40 @@ func isGit(u *url.URL) bool {
 	if strings.ToLower(u.Scheme) == iterator.GitSchemeRaw {
 		return true
 	}
-	if strings.ToLower(u.Scheme) == iterator.HttpsSchemeRaw && strings.ToLower(u.Host) == "github.com" {
+	if strings.ToLower(u.Scheme) == iterator.HttpsSchemeRaw && (strings.ToLower(u.Host) == "github.com" || strings.ToLower(u.Host) == "webrtc.googlesource.com") {
```

Comment:
Cool, but I wonder if we can generalize this for the future. What do you think?

Reply:
Fixed. Added the possible hosts in a list and going through that list to check if the url has one of the hosts.